### PR TITLE
[Docs]Update Kubernetes (Helm)

### PR DESCRIPTION
**Pull request by fengtao1998**
Motivation
Update program output
Make the document easy to read

### CODE REVIEW

**Comment by fengtao1998** [comment]
@Jennifer88huang  PTAL

File: site2/docs/helm-overview.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ It includes support for:
 
 To get up and run with these charts as fast as possible, in a **non-production** use case, we provide a [quick start guide](getting-started-helm.md) for Proof of Concept (PoC) deployments.
 
-This guide walks the user through deploying these charts with default values and features, but *does not* meet production ready requirements. To deploy these charts into production under sustained load, follow the complete [Installation Guide](helm-install.md).
+This guide walks the user through deploying these charts with default values and features, but does not meet production ready requirements. To deploy these charts into production under sustained load, follow the complete [Installation Guide](helm-install.md).
```
Comment (Jennifer88huang) [inline site2/docs/helm-overview.md:56]:
```suggestion
This guide walks you through deploying these charts with default values and features; the default settings do not meet the production ready requirements. To deploy these charts into production under sustained load, follow the [installation guide](helm-install.md).
```

File: site2/docs/helm-overview.md [hunk 2]
```diff
@@ -90,10 +90,10 @@ helm delete <pulsar-release-name>
 ```
 
 For the purposes of continuity, these charts have some Kubernetes objects that cannot be removed when performing `helm delete`.
-It is recommended to *conciously* remove these items, as they affect re-deployment.
+It is recommended to  remove these items, as they affect re-deployment.
```
Comment (Jennifer88huang) [inline site2/docs/helm-overview.md:93]:
```suggestion
It is recommended to remove these items, as they affect re-deployment.
```

File: site2/docs/helm-overview.md (hunk 1 above)
Comment (Jennifer88huang) [reply site2/docs/helm-overview.md:56]: Try to use second person, for details, see https://developers.google.com/style/person

**Review by Jennifer88huang — COMMENTED** [review]
Nice job~
I guess there are more to be refined. If you need some help, we can walk through it and discuss offline

File: site2/docs/helm-deploy.md [hunk 3]
```diff
@@ -341,7 +341,7 @@ Once you have finished the following three things, you can install a Helm releas
 - Install dependent charts.
 - Prepare the Helm release.
 
-In this example, we name our Helm release `pulsar`.
+In this example, the helmet release is named `pulsar`.
```
Comment (Jennifer88huang) [inline site2/docs/helm-deploy.md:344]: helm or helmet?

Comment (fengtao1998) [reply site2/docs/helm-deploy.md]: Sorry, it has been corrected

File: site2/docs/helm-deploy.md [hunk 4]
```diff
@@ -341,7 +341,7 @@ Once you have finished the following three things, you can install a Helm releas
 - Install dependent charts.
 - Prepare the Helm release.
 
-In this example, we name our Helm release `pulsar`.
+In this example, the helm release is named `pulsar`.
```
Comment (sijia-w) [inline site2/docs/helm-deploy.md:344]: the spellings of helm/Helm should be consistent

**Review by Jennifer88huang — COMMENTED** [review]
Concerning the following K8s version, do we need to upgrade k8s server to `v1.16`?  @rockybean
```
kubectl 1.14 or higher, compatible with your cluster ([+/- 1 minor release from your cluster](https://kubernetes.io/docs/tasks/tools/install-kubectl/#before-you-begin))
- Helm v3 (3.0.2 or higher)
- A Kubernetes cluster, version 1.14 or higher
```

**Comment by fengtao1998** [comment]
@Jennifer88huang PTAL

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**